xonsh/tools.py

```diff
@@ -729,9 +729,13 @@ def _make_style(color_name):
     for k, v in _PT_STYLE.items():
         if k in color_name:
             style.append(v)
-    for k, v in _PT_COLORS.items():
+    _custom_colors = builtins.__xonsh_env__.get('PROMPT_TOOLKIT_COLORS', {})
```

Comment:
No need to provide default values here, instead they should be set in the `xonsh.envrion.DEFAULT_VALUES` dict.